docs/best-practices/code-reviews/readme.md

```diff
@@ -1,129 +1,154 @@
 
-  - Reviewers may possess knowledge about the code base that can help
+  - Reviewers may possess knowledge about programming techniques or the code base that can help
     improve or consolidate the change; for example, someone else may be
     concurrently working on a similar feature or fix.
 
+  - Positive Interaction and communication strengthens social bonds between
+    team members.
+
 - **Consistency** in a code base makes code easier to read and
   understand, helps prevent bugs, and facilitates collaboration
   between regular and migratory developer species.
 
 - **Legibility** of code fragments is hard to judge for the author
-  whose brainchild it is, and easy to judge for a reviewer who does
+  whose brain child it is, and easy to judge for a reviewer who does
   not have the full context. Legible code is more reusable, bug-free,
   and future-proof.
 
 - **Accidental errors** (e.g., typos) as well as **structural errors**
-  (e.g., dead code, logic or algorithm bugs) are often much easier to
-  spot for critical reviewers with an outside perspective. (Often,
-  it's a rather satisfying sport to find bugs...)
+  (e.g., dead code, logic or algorithm bugs, performance or architecture
+  concerns) are often much easier to
+  spot for critical reviewers with an outside perspective.
+  (Finding other people's bugs is often a rather satisfying sport ...)
+  Studies have found that even short and informal code reviews have significant
+  [impact on code quality and bug frequency](https://blog.codinghorror.com/code-reviews-just-do-it/).
+
+- **Security reviews** are required in some environments. They are performed
+  by application security engineers following industry-standard
+  procedures and frameworks.
 
-## What to Review
+
+## What to review
 
 There is no eternally true answer to this question and the right
-approach should be agreed on by each development team; some teams prefer
-to review *every* CR merged into the develop branch, while others will have a
+approach should be agreed on by each development team. Some teams prefer
+to review every change merged into the main branch, while others will have a
 "triviality" threshold under which a review is not required. The
 trade-off is between effective use of engineers' (both authors' and
-reviewers') time and maintaining code quality.
+reviewers') time and maintaining code quality. In certain regulatory environments,
+code review may be required even for trivial changes.
+
+Code reviews are classless: being the most senior person on the team does not
+imply that your code does not need review. In the extremely rare case your code
+is actually flawless, less experienced team members can learn from you; in the
+more common case they will point out bugs, suggest newer libraries or techniques,
+or puzzle until they proudly find a more efficient or elegant implementation.
```

Comment:
Think I’d cut this.

Reply:
cut the puzzle? done

Reply:
yeah, I think it might come across as pejorative.